nipype/utils/filemanip.py

```diff
@@ -682,11 +715,21 @@ def read_stream(stream, logger=None, encoding=None):
     return out.splitlines()
 
 
-def savepkl(filename, record):
+def savepkl(filename, record, versioning=False):
     if filename.endswith('pklz'):
         pkl_file = gzip.open(filename, 'wb')
     else:
         pkl_file = open(filename, 'wb')
+
+    if versioning:
+        from nipype import __version__ as version
+        metadata = json.dumps({'version': version},
+                              ensure_ascii=True,
```

Comment:
why not use UTF-8 like we do throughout the package?

Reply:
My initial preference around ASCII over UTF-8, in this case, is about Python 2 retro-compatibility.
If the error contains a path, for example, that uses an "é", Python 2 might not get so happy about it. Since JSON can be fully represented using ASCII, I thought that this config might avoid some headaches.

I've done some tests now, and using UTF-8 will not be a problem at all.